src/topp/MSGFPlusAdapter.cpp

```diff
@@ -181,8 +181,9 @@ class MSGFPlusAdapter :
     registerIntOption_("matches_per_spec", "<num>", 1, "Number of matches per spectrum to be reported (MS-GF+ parameter '-n')", false);
     setMinInt_("matches_per_spec", 1);
 
-    registerFlag_("add_features", "Output additional features - needed e.g. by Percolator (default: basic scores only; MS-GF+ parameter '-addFeatures')", true);
-
+    registerStringOption_("add_features", "<bool>", "true", "Output additional features - needed e.g. by Percolator (default: basic scores only; MS-GF+ parameter '-addFeatures')");
```

Comment:
@timosachsenberg That builds, but is not correct. Could you please point out an example?

Reply:
something like in this snippet:
```
registerStringOption_("bla", "<true/false>", "true", "bla", false, false);
setValidStrings_("bla", ListUtils::create<String>("true,false"));
```

Reply:
also update description to not talk about flag etc.

Reply:
Why? - Can I kick out the exception?